commands/verify/verify.go

```diff
@@ -57,6 +56,31 @@ var Verify = &cobra.Command{
 	},
 }
 
+// verifyInfo contains necessary information about verification
+type verifyInfo struct {
+	caseNumber int
+	retryCount int
+	interval   time.Duration
+	failFast   bool
+	summary    *summary
+	errorExist *bool
```

Comment:
- We do not need put `summary` into `verifyInfo`:
   - Several goroutines may modify it at the same time
   -  we can get those information at last
- `errorExist` is also not needed, we only need to check `errNum` in `summary`